dimod/binary_quadratic_model.py

```diff
@@ -885,6 +923,11 @@ def update(self, bqm):
                 in the updating model are added to the corresponding values in
                 the updated model.
 
+            ignore_info (bool, optional, default=True):
+                If True, the info in the given bqm is ignored, otherwise
```

Comment:
If True, `info` in the given model is ignored; otherwise .... updated with the given model's `info`